file-tests/should-futil/fixed-point-constant.fuse

```diff
@@ -0,0 +1,5 @@
+let a: ufix<4, 2> = (2.25 as ufix<4, 2>); // 9
```

Comment:
What are the numbers after the constants supposed to be? The bitwidth representation? If so, can you add a comment at the top of the test file saying that it is?